test/benchmark/reasoner/iam/complex/ConjunctionStructureTest.java

```diff
@@ -83,7 +83,8 @@ public void testMultipleStartingPoints() {
 
         benchmark.assertAnswerCountCorrect();
         benchmark.assertRunningTime(1000);
-        benchmark.assertCounters(500, 17, 50, 120);
+        benchmark.assertCountersBelow(500, 14, 100, 140, 1.0);
+        benchmark.assertCountersAbove(12, 48, 80, 1.0); // TODO: Assert lower bound for CompoundStreams
```

Comment:
Had to separate the variance was too high to do it in a single number.

The variance is caused indeed by the new scaling, but it seems to be the correct behaviour, since the plans being generated are equal cost as far as the planner can see 
(The specific case is the variables involved in the inter-changeable constraints are bound, hence have estimate 1, so even our multiplicative cost cannot differentiate the plans)